Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java

```diff
@@ -481,22 +477,22 @@ public QueryBuilder getQueryProjectTitle(String title) {
      *            true or false
      * @return list of JSON objects with processes for specific property
      */
-    List<JSONObject> findByProperty(String title, String value, boolean contains) throws DataException {
+    private List<JSONObject> findByProperty(String title, String value, String type, String key, boolean contains) throws DataException {
```

Comment:
You are changing method signature without updating the JavaDoc comment - please update the JavaDoc comment.